python/sparkdl/image/imageIO.py

```diff
@@ -118,89 +82,62 @@ def imageStructToArray(imageRow):
     :param imageRow: Row, must use imageSchema.
     :return: ndarray, image data.
     """
-    imType = imageType(imageRow)
+    imType = imageTypeByOrdinal(imageRow.mode)
     shape = (imageRow.height, imageRow.width, imageRow.nChannels)
     return np.ndarray(shape, imType.dtype, imageRow.data)
 
+def imageStructToPIL(imageRow):
+    imgType = imageTypeByOrdinal(imageRow.mode)
+    if imgType.dtype != 'uint8':
+        raise ValueError("Can not convert image of type " + imgType.dtype + " to PIL, can only deal with 8U format")
+    ary = imageStructToArray(imageRow)
+    if imgType.nChannels != 1:
+        ary = _rgb2bgr(ary)
```

Comment:
I know the function is symmetrical, but shouldn't it be `_bgr2rgb`? The array comes from the row.

Reply:
hmm good point, I'll try to think of a better name, maybe reverseChannels?